### PR TITLE
Add_heatmap_back

### LINKED ISSUE #85: Add_heatmap_back
Add the heatmap that takes the disability into consideration

### PR DESCRIPTION
**Pull request by ThapeloMasasa**
Fixes #85

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -63,8 +67,14 @@ def load_credits_from_yaml(package, resource_path):
...
-children_ages = {
-    num: st.number_input(f"Child {num} Age", 0) for num in range(1, num_children + 1)
+children_ages = {}
+for num in range(1, num_children + 1):
+    age = st.number_input(f"Child {num} Age", min_value=0, max_value=100, step=1, value=0)
+    is_disabled = st.checkbox(f"Child {num} is disabled")
+    children_ages[num] = {'age': age, 'is_disabled': is_disabled}
+disability_status  = {
+    "head": head_disability,
+    "spouse": spouse_disability
```
Comment (PavelMakarchuk) [inline app.py:77]: remove child disability here

Comment (PavelMakarchuk) [reply app.py:77]: and elsewhere

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Comment by PavelMakarchuk** [comment]
Can we validate heatmap results in the spreadsheet as well? Add a column for the heatmap match

File: app.py [hunk 2]
```diff
@@ -357,3 +372,219 @@ def create_table_data(
...
+    return married_situation, single_head_situation, single_spouse_situation
+
+
+def calculate_net_income_for_situation(situation):
+    """
+    Calculate the net income for a given situation using the Simulation class.
+    """
+    simulation = Simulation(situation=situation)
+    net_income = simulation.calculate("household_net_income", YEAR)
+    return np.array(net_income)  # Ensure the result is a numpy array
```
Comment (PavelMakarchuk) [inline app.py:493]: can we store the incomes in pandas dataframes (three total)

File: app.py [hunk 3]
```diff
@@ -70,11 +74,16 @@ def load_credits_from_yaml(package, resource_path):
     "head": head_disability,
     "spouse": spouse_disability
 }
+disability_status  = {
+    "head": head_disability,
+    "spouse": spouse_disability
+}
```
[the remark this answers is not in the document]
Comment (PavelMakarchuk) [reply app.py:80]: duplicate

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Comment by ThapeloMasasa** [comment]
Here is the video for  how it works

https://github.com/PolicyEngine/us-marriage-incentive/assets/73863365/2250b04f-d9bd-4a31-adf3-699eed6c6dcb

**Comment by PavelMakarchuk** [comment]
Can you adjust the labels when hovering over the values? I think we are using the breakdown categories to represent income, e.g.:
image
Here it should say: 
Head Employment income:
Spouse Employment income:
Tax Credit change:

**Pull request merged**